cmd/generate_handlers/generate_handlers_template.txt

```diff
@@ -12,11 +12,11 @@ import (
 
 	"github.com/status-im/status-go/protocol/common"
 	"github.com/status-im/status-go/protocol/protobuf"
-	"github.com/status-im/status-go/messaging"
+	messaging_types "github.com/status-im/status-go/messaging/types"
```

Comment:
I think golang prefers `flatcase` for package names, not `snake_case`:
https://go.dev/blog/package-names#package-names

(as much as I'd dislike it, consistency is more important)

Reply:
Indeed, consistency for the win. Changed to `flatcase`.